easybuild/easyblocks/m/metis.py

```diff
@@ -123,9 +123,22 @@ def sanity_check_step(self):
         if LooseVersion(self.version) < LooseVersion("5"):
             dirs += ["Lib"]
 
+        libext = []
+        if isinstance(self.cfg['configopts'], list):
+            for elements in self.cfg['configopts']:
+                if 'shared=1' in elements:
```

Comment:
METIS makes shared object _or_ archive. Never both. If I use the above mentioned line, I might break old builds, if they would not work with the shared object file. 
I had to give a list in my METIS easyconfig to build both archive and shared object.
```
configopts = [' ', 'shared=1 ']
```